specs/commandline/policy.md

```diff
@@ -0,0 +1,671 @@
+# notation policy
+
+## Description
+
+Use `notation policy` command to add/update/show/delete trust policies. As part of signature verification workflow, user needs to configure the trust policies to specify trusted identities that sign the artifacts, and the level of signature verification to use. For more details, see [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy).
+
+An example of `trustpolicy.json`:
+
+```jsonc
+{
+    "version": "1.0",                                                   // version info
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-dev",                              // Name of the 1st policy
+            "registryScopes": [ "dev.wabbitnetworks.io/net-monitor" ],  // The repository list that policy applies to
+            "signatureVerification": {                                  // The level of verification - strict, permissive, audit, skip
+                "level": "strict"
+                "override" : {
+                     "expiry" : "log",
+                     "authenticity": "log"
+                }
+            },
+            "trustStores": [ "ca:wabbit-networks-dev" ],                // The trust stores that contains the X.509 certificates
+            "trustedIdentities": [                                      // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+            ]
+        },
+        {
+            "name": "wabbit-networks-prod",                             // Name of the 2nd policy.
+            "registryScopes": [ "prod.wabbitnetworks.io/net-monitor" ],       
+            "signatureVerification": {                                
+                "level": "permissive"
+            },
+            "trustStores": [ "ca:wabbit-networks-prod" ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Prod, CN=wabbit-networks.io"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  add           Add trust policies.
+  delete        Delete trust policies.
+  list          List trust policies.
+  update        Update trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy add
+
+```text
+Add trust policies.
+
+Usage:
+  notation policy add [flags] <policy_name>
```

Comment:
If we do ` notation policy add json_file_path`, and there are more than one trust policy statements in the file, they will have different policy names right? It seems that the current design is only taking one <policy_name> as argument. What should we do in this case? According to our original design https://hackmd.io/AP9Xz9AbRpOy6ASpbok23A#Manage-Trust-Policy, we could have something like:
```
Usage:
    notation policy add (--input <policy_file_path> | --name <policy_name>) [flags]
Flags:
   --input string  path to a trust policy file in JSON format
   --name  string  name of the trust policy statement
   ...
```